packages/mobile/src/home/CashInBottomSheet.tsx

```diff
@@ -54,15 +124,35 @@ function CashInBottomSheet() {
         >
           <Times />
         </Touchable>
-        <Text style={styles.title}>{t('cashInBottomSheet.title')}</Text>
-        <Text style={styles.subtitle}>{t('cashInBottomSheet.subtitle')}</Text>
-        <Button
-          text={t('cashInBottomSheet.addFunds')}
-          size={BtnSizes.FULL}
-          onPress={goToAddFunds}
-          style={styles.addFundBtn}
-          testID={'cashInBtn'}
-        />
+        {rampAvailable && rampCashInButtonExpEnabled ? (
+          <>
```

Comment:
Frontend noob question-- is this supposed to be `<>`?  Should it be `<View>`?

Reply:
Its called a Fragment, very similar to a view https://reactjs.org/docs/fragments.html#short-syntax

Reply:
thanks